pkg/windows/windows.go

```diff
@@ -592,10 +592,21 @@ func (vm *windows) createDirectories() error {
 func (vm *windows) removeDirectories() error {
 	vm.log.Info("removing directories")
 	for _, dir := range RequiredDirectories {
+		// exclude WICD binary and credential files removal used in the WICD cleanup cmd
+		if dir == K8sDir {
+			if out, err := vm.Run(rmK8sDirCmd(), true); err != nil {
+				return fmt.Errorf("unable to remove directory %s, out: %s, err: %s", dir, out, err)
+			}
+			continue
+		}
 		if out, err := vm.Run(rmDirCmd(dir), true); err != nil {
 			return fmt.Errorf("unable to remove directory %s, out: %s, err: %s", dir, out, err)
 		}
 	}
+	// cleanup k8sDir, uses best-effort cleanup for WICD service related files
```

Comment:
`Do a best effort deletion of k8sDir which now includes just the WICD related files`